regolith/helpers/u_milestonehelper.py

```diff
@@ -0,0 +1,138 @@
+"""Helper for updating milestones to the projecta collection.
+    It can update the status and due date of a projectum.
+    It can add a new milestone to the projecta collection.
+"""
+import datetime as dt
+from nameparser import HumanName
+import dateutil.parser as date_parser
+import sys
+import uuid
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import all_docs_from_collection
+
+
+TARGET_COLL = "projecta"
+
+def subparser(subpi):
+    subpi.add_argument("projectum_id", help="the id of the projectum")
+    subpi.add_argument("--number",
+                        help="number of the milestone to update from numbered list")
+    subpi.add_argument("--due_date",
+                       help="new due date of the milestone"
+                            "required to add a new milestone")
+    subpi.add_argument("-s", "--status",
+                       help="status of the milestone/deliverable:"
+                            "proposed, started, finished, back_burner"
+                            "paused, cancelled")
+    subpi.add_argument("--name",
+                       help="name of the new milestone."
+                            "required to add a new milestone.")
+    subpi.add_argument("-o", "--objective",
+                       help="objective of the new milestone."
+                            "required to add a new milestone")
+    subpi.add_argument("-t", "--type",
```

Comment:
accept m for meeting, r for release, p for PR and o for other

Reply:
professor, is this what you meant?

Reply:
yes